hwy/targets.h

```diff
@@ -284,6 +291,14 @@ static inline HWY_MAYBE_UNUSED const char* TargetName(int64_t target) {
       HWY_CHOOSE_WASM(func_name),        /* WASM */        \
       nullptr                            /* reserved */
 
+#elif HWY_ARCH_LOONGARCH
+#define HWY_MAX_DYNAMIC_TARGETS 3
+#define HWY_HIGHEST_TARGET_BIT HWY_HIGHEST_TARGET_BIT_LOONG
```

Comment:
There should be HWY_HIGHEST_TARGET_BIT_LOONGARCH?

Reply:
Fixed, thanks.